include/exadg/solvers_and_preconditioners/multigrid/multigrid_preconditioner_base.cpp

```diff
@@ -630,11 +646,14 @@ MultigridPreconditionerBase<dim, Number>::initialize_affine_constraints(
     boundary_functions[it.first] = &zero_function;
   }
 
-  dealii::MappingQ<dim> mapping_dummy(1);
+  auto const & mapping_dummy =
```

Comment:
This has not been introduced in this PR, but maybe we should nevertheless discuss here and then add a comment why we use a dummy mapping here.